PackageExplorer/Converters/UpperCaseConverter.cs

```diff
@@ -8,10 +8,10 @@ public class UppercaseConverter : IValueConverter
     {
         #region IValueConverter Members
 
-        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        public object? Convert(object value, Type targetType, object parameter, CultureInfo culture)
         {
             var s = (string)value;
-            return s.ToUpper(culture);
+            return s?.ToUpper(culture);
```

Comment:
Could this be really null in c# 8?

Reply:
Nullables are currently analysis only. Nothing enforces it, so that's why the null checks are needed.